ic-os/defs.bzl

```diff
@@ -330,6 +332,23 @@ def icos_build(
         ],
     )
 
+    # Disk images just for testing.
+    disk_image_no_tar(
+        name = "disk.img",
+        layout = image_deps["partition_table"],
+        partitions = [
+            "//ic-os/bootloader:partition-esp.tzst",
+            ":partition-grub.tzst",
+            ":partition-boot.tzst",
+            ":partition-root.tzst",
+        ] + custom_partitions,
+        expanded_size = image_deps.get("expanded_size", default = None),
+        tags = ["manual"],
```

Comment:
Are these OK to put into the remote cache?

```
2.5G    ic-os/setupos/envs/dev/disk.img
1.6G    ic-os/hostos/envs/dev/disk.img
1.3G    ic-os/guestos/envs/dev/disk.img
```

Reply:
We don't need to do that -- how do I disable it?

Reply:
I turned it off.